cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/rolling/NodeRoles.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.resource.rolling;
+
+/**
+ * Holds process roles for node
```

Comment:
Can we say how the information here differs from the broker/controller information in a `NodeRef`?